rclcpp/src/rclcpp/event_handler.cpp

```diff
@@ -15,6 +15,7 @@
 #include <stdexcept>
 #include <string>
 
+#include "rcl/error_handling.h"
```

Comment:
Unneeded include ?

Reply:
It's actually a missing include that I added, though to be fair, it's a bit out of scope for the pr. Later in the file functions like `rcl_reset_error()` and types like `rcl_error_state_t`, which come from that header.